tsfile/pom.xml

```diff
@@ -77,6 +77,12 @@
             <groupId>io.airlift</groupId>
             <artifactId>slice</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.tukaani</groupId>
+            <artifactId>xz</artifactId>
+            <version>1.9</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
It also repeats the line 114. Consider removing it?

Reply:
OK， removed it